Libraries/DeprecatedPropTypes/DeprecatedTVViewPropTypes.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ * @format
+ */
+'use strict';
+var PropTypes = require('prop-types');
+
+/**
+ * Additional View properties for Apple TV
+ */
+var DeprecatedTVViewPropTypes = {
+  /**
```

Comment:
You can remove the comments from here so that the source of truth is on the Flow types